### PR TITLE
feat(templates): storyblok

### PR DESCRIPTION
**Pull request by notrab**
# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: templates/graphql-storyblok/grafbase/grafbase.config.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  headers: (headers) => {
+    headers.set('Token', { forward: 'Token' })
+    headers.set('Version', { forward: 'Version' })
+  }
+})
+
+g.datasource(storyblok)
+
+// Disabling namespace may cause conficts with other connectors
+// g.datasource(storyblok, { namespace: false })
```
Comment (fbjork) [inline templates/graphql-storyblok/grafbase/grafbase.config.ts:14]: remove

**Review by fbjork — APPROVED** [review]

File: templates/graphql-storyblok/grafbase/grafbase.config.ts (hunk 1 above)
Comment (notrab) [reply templates/graphql-storyblok/grafbase/grafbase.config.ts:14]: This is inline with all other templates that show how to disable namespacing (all guides mention this too).

Comment (fbjork) [reply templates/graphql-storyblok/grafbase/grafbase.config.ts:14]: Right, but the namespace argument goes in the GraphQL() section

Comment (notrab) [reply templates/graphql-storyblok/grafbase/grafbase.config.ts:14]: With SDL it does yeah

File: templates/graphql-storyblok/grafbase/grafbase.config.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { g, connector, config } from '@grafbase/sdk'
+
+const storyblok = connector.GraphQL('Storyblok', {
+  url: 'https://gapi.storyblok.com/v1/api',
+  headers: (headers) => {
+    headers.set('Token', { forward: 'Token' })
```
Comment (fbjork) [inline templates/graphql-storyblok/grafbase/grafbase.config.ts:6]: don't we want to set the Token via env var?

**Pull request merged**